.gitignore

```diff
@@ -1,5 +1,24 @@
+# Composer Dependencies
 /vendor/*
+/bin
```

Comment:
I guess the question is where are they coming from. 

While they are in the https://github.com/cakephp/app repo, they don't seem to exist in the 3.0 branch of the Cake Core project: https://github.com/cakephp/cakephp/tree/3.0

Did the Cake core just drop them only in the App project but not as scripts in the actual Cake Project?